sda-svc/values.yaml

```
@@ -104,14 +107,16 @@ global:
     useTLS: true
 
   broker:
-    host: ""
+    durable: true
     exchange: ""
-    port: 5671
-    verifyPeer: true
-    vhost: "/"
-# This 
+    host: ""
     password: ""
+    port: 5671
+    routingError: ""
+    ssl: true
     username: ""
+    verifyPeer: false
```

```suggestion
    verifyPeer: true
```
The default should be to use peer verification